packages/assets-controllers/src/assetsUtil.ts

```diff
@@ -292,3 +296,49 @@ export async function reduceInBatchesSerially<
   const finalResult = workingResult as Result;
   return finalResult;
 }
+
+/**
+ * fetches and maps the exchange rates
+ * @param nativeCurrency - native currency
+ * @param tokenAddresses - array of token addresses
+ * @param chainId - The ID of the chain.
+ */
+export async function fetchAndMapExchangeRates(
+  nativeCurrency: string,
+  tokenAddresses: Hex[],
+  chainId: Hex,
+): Promise<ContractExchangeRates> {
+  const tokenPricesFetchService: AbstractTokenPricesService =
+    codefiTokenPricesServiceV2;
+  // check if chainId is supported or currency is supported
```

Comment:
What does this comment say that the code doesn't already say? If the code isn't clear enough, is there a way we can make it clearer?

Reply:
True no need for the comment 🙏